README.md

```diff
@@ -371,4 +373,14 @@ def list_all_data_objects(rule_args, callback, rei):
 
 ### How do I embed single quotes within a string literal?
 
-To embed a single quote character within a string literal, write two adjacent single quotes. For example, `'What''s the current time?'`.
+To embed a single quote character within a string literal, write two adjacent single quotes.
+
+For example, `'What''s the current time?'`.
```

Comment:
let's make an example that is a filename or metadata value - rather than ask a question... that seems misleading for our use case.

Reply:
Do you have something in mind?

Reply:
`'Kory''s file.txt'`

Reply:
Done.